Unity-Technologies/ScriptableRenderPipeline
# Adding Linux Pipeline CI for SRP

### Purpose of this PR
Adding yamato pipeline tests for Linux for LWRP, PostProcessing, and Shadergraph.  I also added Linux pipeline support for HDRP, vfx_lwrp, and vfxmain.  I also made one small change to fix an issues with capitalization in path names when running graphics tests in Linx Standalone
---
### Release Notes
These are passing for the most part except for some real failures for LWRP playmode and LWRP standalone seems to crash when building, but only on the bokken VM.  Locally, I cannot reproduce the issue.   HDRP, vfx_lwrp, and vfxmain have also had the pipeline added, but they are not yet stable.

---
### Testing status
**Yamato Tests**:  

LWRP - Editor, Playmode are passing.  Standalone crashes on building the project on the Bokken VM.  Passes locally.

Post Processing an Shadergraph are all passing

HDRP, vfx_lwrp, and vfxmain do not pass yet.  The pipelines are valid, but the projects may need to be modified for work with Linux.

**Manual Tests**: What did you do?
- [ X] Opened test project + Run graphic tests locally
- [X ] Built a player
- [N/A ] Checked new UI names with UX convention
- [N/A ] Tested UI multi-edition + Undo/Redo
- [N/A ] C# and shader warnings (supress shader cache to see them)
- [N/A] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Automated Tests**:
Added OpenGL Linux tests for yamato pipelines.

Launch Katana (Link on master here - update for your branch):
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=master&automation-tools_branch=master&unity_branch=trunk&sort=1-asc

Alternative, launch Yamato (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None

**Halo Effect**: None

---
### Comments to reviewers
